src/pages/Components/FileUploader/code.md

```diff
@@ -2,40 +2,54 @@
 title: 'Code'
 order: 2
 ---
-
 ## Import
 
+Import the settings and the checkbox `scss` files.
+
 ```css
 @import 'settings-tools/_all-settings';
-@import '...';
+@import 'components/_c.file-uploader';
 ```
 
 ## Basic usage
+> Natively, to upload files we use an `<input type="file"/>`, but in Mozaic, we will customize it with a `<label>` that will act as a button.
 
-Apply the classes...
+Wrap a input[type="file"] and a label inside a div and apply the following classes :
 
 ```html
```

Comment:
I think it's use less because you have the preview just below

Reply:
I don't understood, what do you mean ?

Reply:
I was talking about the html example just before the preview example which also have an html preview ;)

Reply:
ah, you speak about html quote ?

Reply:
yes ;)